lib/matplotlib/axes/_axes.py

```diff
@@ -1408,59 +1408,67 @@ def plot(self, *args, **kwargs):
     def plot_date(self, x, y, fmt='o', tz=None, xdate=True, ydate=False,
                   **kwargs):
         """
-        Plot with data with dates.
-
-        Call signature::
-
-           plot_date(x, y, fmt='bo', tz=None, xdate=True,
-                     ydate=False, **kwargs)
-
-        Similar to the :func:`~matplotlib.pyplot.plot` command, except
+        A plot with data that contains dates.
+        
+        Similar to the :func:~matplotlib.pyplot.plot command, except
         the *x* or *y* (or both) data is considered to be dates, and the
         axis is labeled accordingly.
 
         *x* and/or *y* can be a sequence of dates represented as float
         days since 0001-01-01 UTC.
 
-        Keyword arguments:
+        Note if you are using custom date tickers and formatters, it
+        may be necessary to set the formatters/locators after the call
+        to meth : plot_date since meth : plot_date will set the
```

Comment:
The backticks should not have been removed here and the spaces around the colons are not necessary.